config/application.rb

```diff
@@ -7,12 +7,15 @@
 require_relative '../lib/rack_x_robots_tag'
 
 # Require the gems listed in Gemfile, including any gems
-# you've limited to :test, :development, or :production.
+# you've limited to :test, : elopment, or :production.
```

Comment:
nit: Was this auto generated? Just wondering why it is "elopment" vs "development"

Reply:
good catch, I must have accidentally deleted "dev" somehow